lib/config/definitions.js

```diff
@@ -597,12 +597,14 @@ const options = [
   },
   {
     name: 'assignees',
-    description: 'Assignees for Pull Request',
+    description:
+      'Assignees for Pull Request (with email in vsts, user name in GitHub)',
     type: 'list',
   },
   {
     name: 'reviewers',
-    description: 'Requested reviewers for Pull Requests (GitHub only)',
+    description:
+      'Requested reviewers for Pull Requests (with email or user name in vsts, user name in GitHub)',
```

Comment:
So assignees needs to be an email, but reviewers can be email or username?

Reply:
Yes. Actually in VSTS you have a clear field for reviewers.
For assignees, I just create a comment with @<emailAddress> (this is the way to tag someone. Of course, when you do this in VSTS UI directly, you have autocompletion with the name)

I think it's the best we can do here